configs/sandpack-contents/component-theming/heading.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+  App: `import { Box, SimpleGrid, IconButton, Center, Heading, useColorMode } from "@chakra-ui/react";
+import { FaMoon, FaSun } from "react-icons/fa";
+export default function App() {
+    const { toggleColorMode, colorMode } = useColorMode();
+    return (
+        <Box position="relative" h="100vh">
+            <SimpleGrid gap={12} p={12} columns={2}>
+              <Heading>Default heading</Heading>
+              <Heading variant="custom">Themed heading</Heading>
+              <Heading variant="brand">Branded heading</Heading>
+              <Heading variant="underline">Underlined heading</Heading>
+            </SimpleGrid>
+            <IconButton
+                rounded="full"
```

Comment:
The `IconButton` requires an `aria-label` prop. Put this as the first prop: `aria-label="toggle theme"`